code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -102,7 +102,7 @@
 		return TRUE
 
 /obj/structure/bed/attackby(obj/item/weapon/W as obj, mob/user as mob)
-	if(istype(W, /obj/item/weapon/tool/wrench))
+	if(QUALITY_BOLT_TURNING in W.tool_qualities)
```

Comment:
We have procs for this, things like has_quality
Make use of them rather than checking vars directly

Reply:
Done.